app/components/products/ChipActionBanner.tsx

```
@@ -0,0 +1,222 @@
+    const theme = useTheme();
+    const holdersAccStatus = useHoldersAccountStatus(address).data;
+    const isDogs = getDogsRef();
+
+    let bannerStatus: BannerStatus = 'not-active';
+
+    switch (holdersAccStatus?.state) {
+        case HoldersUserState.NeedEmail:
+            bannerStatus = 'email';
+            break;
+        case HoldersUserState.NeedKyc:
+            const kyc = holdersAccStatus.kycStatus;
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)